src/generator.ts

```diff
@@ -343,7 +343,7 @@ function generateStandaloneInterface(ast: TNamedInterface, options: Options): st
     (hasComment(ast) ? generateComment(ast.comment) + '\n' : '') +
     `export interface ${toSafeString(ast.standaloneName)} ` +
     (ast.superTypes.length > 0
-      ? `extends ${ast.superTypes.map(superType => toSafeString(superType.standaloneName)).join(', ')} `
+      ? `extends ${ast.superTypes.map(superType => toSafeString(superType.standaloneName || '')).join(', ')} `
```

Comment:
Default needed here because `CustomType` doesn't have the `standaloneName` property thus by making `ast.superTypes` return an array of `TNamedInterface | TCustomType` standaloneName can now be undefined as well